src/views/pages/GenerateKeypair/index.js

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
+import Button from 'react-bootstrap/Button';
+import CLI from '../../../services/CLI';
+import { CLI_URI } from '../../../config';
+import { Client as WsClient } from 'rpc-websockets';
 
 function GenerateKeypair() {
+
+  async function generateKeypair() {
```

Comment:
@Romulus10 I think I'm not quite passing the param correctly?

Reply:
Ah I think I see what's happening - when you run the fennel-cli with `start-rpc` it'll expose a websocket that you can interact with fairly similarly to how you'd interact with a REST API. Rather than calling the binary with an argument, it'd be a message accessing the RPC's generate-keypair method. I'll add it to the top of my backlog to work out a better example for that with WSClient.